src/state/network/actions.js

```diff
@@ -18,14 +18,23 @@ export function makeApiRequest(requestId, root, endpoint, options = {}) {
     const api = new APIClient(root, accessToken);
     const request = getRequest(state, requestId);
 
+    // A "stealth API request" is one that doesn't update the state.
+    // This is useful when you generally "don't care" about the *state* of the request.
+    const stealth = options.stealth || false;
+    if (stealth) {
+      // Doing it this way because JavaScript doesn't have a `obj.pop(key, default)`
+      delete options.stealth;
+    }
+
     if (request.inProgress) {
       return true;
     }
 
-    dispatch({
-      type: REQUEST_SEND,
-      requestId,
-    });
+    !stealth &&
```

Comment:
Can we write this as `if (!stealth) { ... }` instead? `&&` version is useful when you need an expression, but we don't need an expression here.